hack/cluster-openshift.sh

```diff
@@ -512,6 +520,13 @@ if [ "$_CMD" = "up" ]; then
     KIALI_PASSPHRASE=${KIALI_PASSPHRASE} sh
   fi
 
+  if [ "${REMOVE_JAEGER}" == "true" ]; then
+      echo "Removing Jaeger from cluster..."
+      ${MAISTRA_ISTIO_OC_COMMAND} delete all,secrets,sa,templates,configmaps,deployments,clusterroles,clusterrolebindings,virtualservices,destinationrules --selector=app=jaeger -n istio-system
```

Comment:
Side comment.
Perhaps our colleagues @objectiser or @pavolloffay would like to know the problems with Elasticsearch.
Just in case if there is some config parameter that could help.

To Gary/Pavol: this is just a hack script for developer when playing with a local installation. No other intention here.

Reply:
@lucasponce to what problems with ES are you referring to?